token/core/cmd/pp/common/common.go

```diff
@@ -68,3 +86,60 @@ func SetupIssuersAndAuditors(pp PP, Auditors, Issuers []string) error {
 	}
 	return nil
 }
+
+func ReadFile(file string) ([]byte, error) {
+	fileCont, err := ioutil.ReadFile(file)
+	if err != nil {
+		return nil, errors.Wrapf(err, "could not read file %s", file)
+	}
+
+	return fileCont, nil
+}
+
+func ReadPemFile(file string) ([]byte, error) {
```

Comment:
Let's give the function some documentation and even change the name. `ReadPemFile` is a very generic name, however, the current impl returns errors if the content in the pem file it not a cert, or there are more than one entry.

Maybe call it `ReadCertFromPemFile(file string)` explaining that it returns the first item from a pem file if exists. If needed we can later add `ReadCertsFromPemFile(...)` to parse the entire file.

Reply:
okay